src/System/Taffybar.hs

```diff
@@ -175,12 +182,15 @@ data TaffybarConfig =
                  , endWidgets :: [IO Widget] -- ^ Widgets that are packed from right-to-left in the bar
                  }
 
+type TaffybarConfigEQ = (TaffybarConfig, StableName TaffybarConfig)
```

Comment:
> I assume you are using the StableName for equality tests later. Is it possible to just do comparisons on (screenNumber, monitorNumber) instead?

No, because the TaffybarConfiguration that is being provided for the monitor could also change. This is an important feature to me, at least for the time being, because the tray widget can only be placed on 1 monitor, and I need to have the ability to move this around (when I plug in/unplug my laptop).